scripts/deploy.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+const {argv, env, exit} = require('process');
+const {execSync} = require('child_process');
+
+const {loadEnv} = require('./common');
+
+const stage = argv[2];
+if (!stage) {
+    console.error("Usage: deloy.js <stage name>")
+    exit(1);
+}
+
+const envPath = `.${stage}.env`;
+loadEnv(envPath);
+
+const bucket = env['S3_BUCKET'];
```

Comment:
Would you also add a note in the `README.md` file on the variables in the `.production.env` file and `.staging.env` file and how to find them via AWS dashboard? I guess `*.env` files are just text files that have `ENVNAME=foobar` per line?

Reply:
OK, my last question is already answered by the dotenv's [docs](https://github.com/motdotla/dotenv#readme).

Reply:
:+1: